examples/gno.land/p/demo/blog/util.gno

```diff
@@ -5,3 +5,26 @@ import "strings"
 func breadcrumb(parts []string) string {
 	return "# " + strings.Join(parts, " / ") + "\n\n"
 }
+
+func (p postSlice) GetPost(slug string) *Post {
+	for _, post := range p {
+		if post.Slug == slug {
+			return post
+		}
+	}
+	return nil
+}
+
+// Sort interface implementation for postSlice
+
+func (p postSlice) Len() int {
+	return len(p)
+}
+
+func (p postSlice) Less(i, j int) bool {
+	return p[i].CreatedAt.Before(p[j].CreatedAt)
+}
+
+func (p postSlice) Swap(i, j int) {
+	p[i], p[j] = p[j], p[i]
+}
```

Comment:
```suggestion
}

var _ sort.Interface = postSlice(nil)
```

This doesn't do anything code-wise, but it documents to the reader "this type implements sort.Interface" explicitly, and it breaks at precompile (or compile stage in Go) if for some reason postSlice is changed not to implement sort.Interface